cmd/apps.go

```diff
@@ -85,8 +86,14 @@ func AppsList(results int) error {
 
 	fmt.Printf("=== Apps%s", limitCount(len(apps), count))
 
+	var appNames []string
```

Comment:
Instead of converting them into a slice of strings with their names, why don't we provide a new type called `Apps` in the SDK and provide the following functions to the [Apps struct](https://github.com/deis/controller-sdk-go/blob/cd8ecc1f17db726e57586a4d149b60a25f3c4dc7/api/apps.go#L3-L11)?

```
type Apps []App

func (a Apps) Len() int           { return len(a) }
func (a Apps) Swap(i, j int)      { a[i], a[j] = a[j], a[i] }
func (a Apps) Less(i, j int) bool { return a[i].ID < a[j].ID }
```

That way we can make `apps.List` return type `Apps`, which can then use `sort.Sort(apps)` without having to perform all this logic ourselves to generate a sort-able list.

Either way LGTM, just explaining a "best practices" approach to this problem. :)